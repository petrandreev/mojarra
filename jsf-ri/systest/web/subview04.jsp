<!--
Copyright 2004 Sun Microsystems, Inc. All rights reserved.
SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
    <html>
    <head>
        <title>subview04</title>
    </head>

    <body>
    <p>
        <h:outputText value="[A]"/></p>

    <f:subview id="foo02">
        <p><h:outputText value="Begin test < c:include> with subview tag in
            including page"/></p>

        <p>
            <jsp:include page="bar01.jsp"/>
        </p>
    </f:subview>

    <p><h:outputText value="subview04"/></p>

    <f:subview id="bar02">
        <p>
            <jsp:include page="bar02.jsp"/>
        </p>
    </f:subview>

    <p><h:outputText value="End test < c:include> with subview tag in including
        page"/></p>
    </body>
    </html>
</f:view>
