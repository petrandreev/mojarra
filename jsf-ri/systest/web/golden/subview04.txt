<!--
 Copyright 2003 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->






<html>
<head>
<title>subview04</title>
</head>
<body>
[A]



subview04

bar02<!--
 Copyright 2003 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->










End test &lt;c:include&gt; with subview tag in including page
</body>
</html>

